encode/av1encode.c

```diff
@@ -1856,12 +1857,6 @@ pack_render_size(bitstream* bs)
     uint32_t render_and_frame_size_different = 0;
 
     put_ui(bs, render_and_frame_size_different, 1);//render_and_frame_size_different
-
-    if (render_and_frame_size_different == 1)
```

Comment:
Can we keep this part of code? This is from av1 spec. Although we do not fully support customize all parameters, but suggesting to keep to code here for further extension.

Reply:
Have consulted with Carl regarding the dead code, and it can be removed from the codebase.